broker/src/tunneldigger_broker/tunnel.py

```diff
@@ -330,6 +330,7 @@ def message(self, address, msg_type, msg_data, raw_length):
         if msg_type == protocol.CONTROL_TYPE_ERROR:
             # Error notification from the remote side.
             # TODO: Parse error code.
+            logger.warning("Tunnel %d (%s) got error from remote peer", self.tunnel_id, self.uuid)
```

Comment:
What about parsing the error code and logging it?

Reply:
I mean, that would be nice, but for now having the code printed is better than nothing I think.